### PR TITLE
Updates some salvage lore

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
Title. Updates some lore, makes it more accurate and flow better.

## Why It's Good For The Game
lore accuracy good. description flow good.

## Testing
- loaded
- spawned salvage
- observed salvage
- success

<hr>

### Declaration
- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/feature_freeze/CODE_OF_CONDUCT.md#currently-changes-to-the-following-types-of-content-reuires-pre-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.

<hr>

## Changelog
:cl:
tweak: changed some salvage lore
/:cl:

### CODE REVIEW

File: code/game/objects/items/salvage.dm [hunk 1]
```diff
@@ -75,7 +75,7 @@
 /// Loot salvage, gained from fighting space simplemobs.
 
 /obj/item/salvage/loot
-	desc = "A tonne of salvage looted from a fallen foe. Who spawned the base type? Report this on the github."
+	desc = "A tonne of salvage looted from a fallen foe. Would probably be pretty valuable, if you could figure out what the hell it was. Who spawned the base type? Report this on the github."
```
Comment (Fordoxia) [inline code/game/objects/items/salvage.dm:78]:
```suggestion
	desc = "If you can see this forbidden salvage, report it on GitHub."
```

File: code/game/objects/items/salvage.dm [hunk 2]
```diff
@@ -58,15 +58,15 @@
 
 /obj/item/salvage/ruin/carp
 	name = "carp scales"
-	desc = "A collection of scales shed from a corrupted space carp. Their culinary potential could mean untold riches for Nanotrasen."
+	desc = "A collection of scales shed from a corrupted space carp. Their uniquie molecular composition may prove useful to material scientists."
```
Comment (Taurtura) [inline code/game/objects/items/salvage.dm:61]:
```suggestion
	desc = "A collection of scales shed from a corrupted space carp. Their unique molecular composition may prove useful to material scientists."
```

File: code/game/objects/items/salvage.dm [hunk 3]
```diff
@@ -58,15 +58,15 @@
...
 	icon_state = "dragon_scales"
 	hitsound = sound('sound/effects/hit_on_shattered_glass.ogg', 20)
 	pickup_sound = sound('sound/hallucinations/im_here2.ogg', 10)
 	drop_sound = sound('sound/hallucinations/look_up2.ogg', 10)
 
 /obj/item/salvage/ruin/tablet
 	name = "mysterious tablet"
-	desc = "A mysterious and old stone tablet. When you read the text on it, you start getting chills."
+	desc = "A old, weathered tablet made of dark stone. Merely looking at it sends chills down your spine."
```
Comment (Taurtura) [inline code/game/objects/items/salvage.dm:69]:
```suggestion
	desc = "An old, weathered tablet made of dark stone. Merely looking at it sends chills down your spine."
```

**Review by Taurtura — CHANGES_REQUESTED** [review]

File: code/game/objects/items/salvage.dm [hunk 4]
```diff
@@ -75,7 +75,7 @@
 /// Loot salvage, gained from fighting space simplemobs.
 
 /obj/item/salvage/loot
-	desc = "A tonne of salvage looted from a fallen foe. Who spawned the base type? Report this on the github."
+	desc = "If you can see this forbidden salvage, report it on GitHub."
```
Comment (lewcc) [inline code/game/objects/items/salvage.dm:78]: aww, I kinda like this one

Comment (Fordoxia) [reply code/game/objects/items/salvage.dm:78]: My main concern was that it's very verbose for something that you do not want to appear, ever.

Priority is communicating with the player that they should report, and that shouldn't be diluted IMHO.

**Review by lewcc — APPROVED** [review]
Looks good to me

**Review by Burzah — APPROVED** [review]

**Pull request merged**